frontend/src/metabase/writeback/components/ActionCreator/ActionCreator.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { t } from "ttag";
+import { connect } from "react-redux";
+
+import type { ActionType } from "metabase/writeback/types";
+
+import { ActionCreatorHeader } from "./ActionCreatorHeader";
+import { QueryActionEditor } from "./QueryActionEditor";
+import { FormCreator } from "./FormCreator";
+
+import {
+  ActionCreatorRoot,
+  ActionCreatorBodyContainer,
+} from "./ActionCreator.styled";
+
+import { getMetadata } from "metabase/selectors/metadata";
+
+import type { State } from "metabase-types/store";
+import type Question from "metabase-lib/lib/Question";
+
+import { newQuestion } from "./utils";
+import SaveActionModal from "./SaveActionModal";
```

Comment:
Let's keep the local imports at the bottom of the imports list. [See the style guide](https://www.metabase.com/docs/latest/developers-guide/frontend.html#style-guide)